src/Common/tests/System/Diagnostics/RemoteExecutorTestBase.cs

```diff
@@ -139,6 +137,7 @@ private static RemoteInvokeHandle RemoteInvoke(MethodInfo method, string[] args,
             }
             else
             {
+                //Desktop activation
```

Comment:
The `if` check above doesn't really make sense anymore. It essentially checks if the process's main module exists, which it necessarily must. Did we determine if this is actually used anywhere outside of the .NET Core tests?